secure-review/apps/quarkus-conference/speaker/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
We are relying directly on installed Maven, so we can remove completely the .mvn folder.